### PR TITLE
Create upgrade_partner_service

### PR DESCRIPTION
**Pull request by iritch42**

### CODE REVIEW

**Comment by thinkingserious** [comment]
Hello @iritch42,<br /><br/>Thank you for the PR!<br />                                              <br />We will be reviewing this PR soon, in the mean                                              time, please be sure to                                              <a href="https://github.com/sendgrid/docs/blob/master/CONTRIBUTING.md#cla">                                             sign our CLA</a> so that we can merge your changes. Thanks again!                                             <br /><br />Team DX

File: source/Classroom/Basics/Billing/upgrade_partner_service [hunk 1]
```diff
@@ -0,0 +1,25 @@
+v —
+st:
+  published_at: 2016-05-26
+  type: Classroom
+seo:
+  title: How do I upgrade my account if I am using SendGrid through a parter service?
```
Comment (ksigler7) [inline source/Classroom/Basics/Billing/upgrade_partner_service]: partner is spelled wrong

File: source/Classroom/Basics/Billing/upgrade_partner_service [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  description: Need to upgrade your SendGrid account?
+  keywords: upgrade, account, third, party, service, Azure, Heroku, KKE, Google, Softlayer, AppDirect
+title: How do I upgrade my account if I am using SendGrid through a parter service? 
+weight: 0
+layout: page
+navigation:
+  show: true
+---
+
+If you wish to use upgrade an account purchased through one of our third party partners, you must upgrade your SendGrid service through the partner service and not through the SendGrid site or interface.
```
Comment (ksigler7) [inline source/Classroom/Basics/Billing/upgrade_partner_service]: remove use

**Comment by ksigler7** [comment]
hey @iritch42 if you could please fix the typos I pointed out in my comments, we can move forward with merging this!

**Comment by ksigler7** [comment]
thanks @iritch42 !

**Pull request merged**